com.intellij.diagnostic.PluginException: Fatal error initializing 'com.intellij.plugin.bootstrap.BootstrapPluginApplicationComponent            ' [Plugin: nl.jworks.intellij.bootstrap3]    at com.intellij.serviceContainer.ComponentManagerImpl.handleInitComponentError$intellij_platform_serviceContainer(ComponentManagerImpl.kt:571)    at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:382)    at com.intellij.serviceContainer.ComponentManagerImpl.access$registerComponents(ComponentManagerImpl.kt:50)    at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:260)    at com.intellij.openapi.client.ClientAwareComponentManager.registerComponents(ClientAwareComponentManager.kt:46)    at com.intellij.idea.ApplicationLoader.doInitApplication(ApplicationLoader.kt:78)    at com.intellij.idea.ApplicationLoader.access$doInitApplication(ApplicationLoader.kt:1)    at com.intellij.idea.ApplicationLoader$doInitApplication$1.invokeSuspend(ApplicationLoader.kt)    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)    at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)    at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)    at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)    at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)    at com.intellij.idea.ApplicationLoader.initApplication(ApplicationLoader.kt:61)    at com.p.s.k.p._g.k(_g.java:60)    at com.p.s.k.p.p6.k(p6.java:113)    at com.p.s.k.p.pa.p(pa.java:139)    at com.p.s.k.p.pG.p(pG.java:57)    at com.intellij.idea.MainImpl.start(MainImpl.kt:13)    at com.intellij.idea.StartupUtil$startApplication$7.invokeSuspend(StartupUtil.kt:294)    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)    at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)    at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)    at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)    at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)    at com.intellij.idea.Main.main(Main.kt:36)Caused by: java.lang.ClassNotFoundException: com.intellij.plugin.bootstrap.BootstrapPluginApplicationComponent             PluginClassLoader(plugin=PluginDescriptor(name=Bootstrap 3, id=nl.jworks.intellij.bootstrap3, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/IdeaIC2022.3/plugins/bootstrap3.jar, version=4.1.1, package=null, isBundled=false), packagePrefix=null, instanceId=12, state=active)    at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:215)    at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)    at com.intellij.serviceContainer.ComponentManagerImpl.registerComponent(ComponentManagerImpl.kt:495)    at com.intellij.serviceContainer.ComponentManagerImpl.registerComponents(ComponentManagerImpl.kt:374)    ... 26 more-----Your JRE: 17.0.5+1-b653.23 aarch64 (JetBrains s.r.o.)/Applications/IntelliJ IDEA CE.app/Contents/jbr/Contents/Home```